### DISCUSSION TITLE
Already merged code not visible opening new pull request

### DISCUSSION THREAD

**Discussion by krogenlundVisma** [community/community#34577] [Pull Requests]
I create a pull request in the web interface e.g. development to master and merge it. Then I do the exact same pull request and the same diff comes up - making it impossible to see if there are any diff. If I merge the second pull request the history states a merge with zero diff. Are there different ways to compare branches and is this really the ways this should work?
Working with bitbucket and TFS this never happened. 

Thank you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @krogenlundVisma ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by krogenlundVisma** [reply]
Did not help. Okay so if this is the way Github works(!) is the workflow-strategy then not updating you feature branch from master or development. This would avoid non-changes from showing up. Im about to tell Elon about this matter!
